### PR TITLE
DropDown

### PR DESCRIPTION
**Pull request by butalas**
### Description
Creates the DropDown component

### Context
Archita and I created the DropDown component using the `Picker` in ReactNative. We incorporated it using the arrow since a user will always be pressing that arrow to trigger the `Picker`. We implemented it in the Item Page, since that page is functional. 

### Issue
#92 

### Screenshot
simulator screen shot - iphone x - 2019-03-01 at 16 10 58

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ const driverStackNavigator = createStackNavigator({
 
 const TabNavigator = createBottomTabNavigator({
   ShopSearch: shopStackNavigator,
-  Notifications: Notifications,
+  Notifications: Item,
```
Comment (ijemmao) [inline App.js]: Change this back to `Notifications`

File: components/DropDown.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <TouchableOpacity onPress={() => this.togglePicker()}>
+            <Image style={styles.downArrowIcon}
+                source={require('./../assets/images/Vector.png')} />
+        </TouchableOpacity>
+        <Modal visible={this.state.pickerDisplayed} animationType={"slide"} transparent={true}>
+            <Picker
+            style={{ backgroundColor: '#f9f9f9', position: 'absolute', bottom: 0, left: 0, right: 0 }}
+            selectedValue={ this.state.pickerSelection }
+            onValueChange={(itemValue, itemIndex) => this.setState({ pickerSelection: itemValue})}>
+            <Picker.Item label="1" value="1" />
```
Comment (ijemmao) [inline components/DropDown.js]: Use a function to dynamically generate these components

**Comment by ijemmao** [comment]
@butalas Did you get a chance to complete this?

**Comment by butalas** [comment]
I've had something else during the day, so I'm only just taking a look at it again, but I'm kind of stuck on the passing a function into the component part. It won't let me pick when the function is being called. It just always goes back to the first item of the picker. Everything else is working though.

**Comment by butalas** [comment]
Hey! The component is finally done and linked with the item page so the quantity actually changes on screen! Here's a pic: 

simulator screen shot - iphone x - 2019-03-05 at 18 56 56

**Comment by ijemmao** [comment]
@butalas This is looking amazing! Just resolve your conflicts as I mentioned during the meeting

**Comment by butalas** [comment]
Sorry I accidentally closed the PR! Just resolved the conflicts!

**Pull request merged**